zp-relayer/configs/relayerConfig.ts

```diff
@@ -25,6 +26,7 @@ const config = {
   relayerPrivateKey: process.env.RELAYER_ADDRESS_PRIVATE_KEY as string,
   tokenAddress: process.env.RELAYER_TOKEN_ADDRESS as string,
   relayerGasLimit: toBN(process.env.RELAYER_GAS_LIMIT as string),
+  minBaseFee: toBN(process.env.RELAYER_MIN_BASE_FEE || '0'),
```

Comment:
Let's add all new variables to CONFIGURATION.md

Reply:
Updated